hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/DiskBalancerCommands.java

```diff
@@ -43,13 +44,16 @@
  *        datanodes
  *      ozone admin datanode diskbalancer start -a
  *        start balancer with default values in the configuration on all
- *        datanodes in the cluster
+ *        datanodes in the cluster and stops automatically after balancing
  *      ozone admin datanode diskbalancer start -t 5 -d {@literal <hosts>}
  *        start balancer with a threshold of 5%
  *      ozone admin datanode diskbalancer start -b 20 -d {@literal <hosts>}
  *        start balancer with maximum 20MB/s diskbandwidth
  *      ozone admin datanode diskbalancer start -p 5 -d {@literal <hosts>}
  *        start balancer with 5 parallel thread on each datanode
+ *      ozone admin datanode diskbalancer start -s=false -a}
+ *        start balancer and will keep running until stopped by the
```

Comment:
start balancer on each datanode and will keep running even after disks are balanced until stopped by the stop command.